wall_e/src/resources/cogs/ban.py

```diff
@@ -100,18 +100,24 @@ async def intercept(self, guild: discord.Guild, member: Union[discord.User, disc
 
         # need to read the audit log to grab mod, date, and reason
         logger.info(f"[Ban intercept()] guild ban detected and intercepted for user='{member}'")
+        logger.info(f"[Ban intercept()] waiting 1 second to ensure ban log is created")
+        await asyncio.sleep(1)
```

Comment:
please add a comment here about why this sleep is needed.

I know its in your PR but having it in the code is a good idea as well so someone does not have to bring up github to find the reason why its here.